code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -385,14 +385,21 @@
 		if(src.loc == summoner)
 			to_chat(src, "<span class='danger'><B>You must be manifested to heal!</span></B>")
 			return
+		if(issythetic(target))
+			src.changeNext_move(CLICK_CD_MELEE)
+			if(heal_cooldown <= world.time && !stat)
+				var/mob/living/synthetic/I = target
+				if (I && I.damage >= 0)
+					I.damage = -10
+					heal_cooldown = world.time + 20
```

Comment:
Nope, there isn't a ```/mob/living/synthetic``` which is why travis has failed the checks.

Welcome to the code. First lesson, always compile and test your code before pushing it :wink: